xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/AspspCsvServiceImpl.java

```diff
@@ -56,6 +62,22 @@ public void importCsv(byte[] bytes) {
         aspspRepository.saveAll(aspsps);
     }
 
+    @Override
+    public void saveCsv() {
+        try {
+            if (CSV_ASPSP_ADAPTER_CONFIG_FILE_PATH == null || CSV_ASPSP_ADAPTER_CONFIG_FILE_PATH.isEmpty()) {
+//                String filePath = getClass().getClassLoader().getResource(DEFAULT_CSV_ASPSP_ADAPTER_CONFIG_FILE).getPath();
+//                Files.write(Paths.get(filePath), exportCsv());
+                throw new RegistryIOException("Adapter config file path is not set or Configuration File does not exist");
```

Comment:
"Adapter config file" to me refers to `adapter.config.properties`. Also what "Configuration File" may not exist here and how did you determine that? It's not an IOException.